docs/kb/semgrep-appsec-platform/saml-stops-working.md

```diff
@@ -8,92 +8,31 @@ tags:
 
 import MoreHelp from "/src/components/MoreHelp"
 
-# SAML stops working
+# Troubleshooting SAML SSO
 
-This article walks you through troubleshooting categorical SAML failures, or
-instances where your SAML configuration stops working suddenly after you've
-successfully configured it and used it for some time. There are several reasons
-why these failures happen.
+This article walks you through troubleshooting SAML SSO failures, including the case where your SAML configuration stops working after you've successfully configured it and used it for some time. There are several common reasons why a configuration may fail.
 
-## The SAML configuration uses a faulty SSL certificate
+## Certificate issues
 
-One component of the SAML configuration is host validation of the targeted IdP
-server, which entails uploading and registering the IdP's x509 certificate. If
-the certificate becomes invalid - expiration, revocation, or rotation - host
-validation fails for the IdP server. Then, the IdP server fails to respond to
-requests.
+If you see the error `Signature validation failed. SAML Response rejected` it's likely there is a problem with the certificate. If the IdP certificate becomes invalid via expiration, revocation, or rotation, signature validation fails.
```

Comment:
```suggestion
If you see the error `Signature validation failed. SAML Response rejected` it's likely there is a problem with the certificate. The IdP certificate may be invalid due to expiration, revocation, or rotation, or signature validation fails.
```

Reply:
This is not semantically the same as what I said - the signature validation failure is a result of the other issues, not a cause of an issue. Skipping this one, feel free to adjust the wording later as long as the sense is preserved.

Reply:
That's probably fine for this context. It fails if the cert becomes invalid for any reason, but those are the most common reasons. I'll make the change.